app/src/main/java/io/github/trojan_gfw/igniter/TrojanConfig.java

```diff
@@ -79,6 +81,7 @@ public String generateTrojanConfigJSON() {
             return new JSONObject()
                     .put("local_addr", this.localAddr)
                     .put("local_port", this.localPort)
+                    .put("remote_server_name", this.remoteServerName)
```

Comment:
This doesn't belong to Trojan config.

Reply:
This field only mark the remote address in order to remember, which like the field in file, gui-config.json, download from the website.

Reply:
> This doesn't belong to Trojan config.

This field only mark the remote address in order to remember, which like the field in file, gui-config.json, download from the website. What do you think?

Reply:
How about renaming it to ServerRemark?

Reply:
NP